### ISSUE TITLE
Emscripten build is just too slow

### ISSUE THREAD

**Issue by yksoft1** [joncampbell123/dosbox-x#900]
**Describe the bug**
Emscripten build is just too slow for pratical use.

**To Reproduce**
use Autoconf-based build or static Makefiles to build Dosbox-X with Emscripten, and make HTML/JS files.
Deploy the generated files in a Web server and run the generated HTML in a Web browser.

**Expected behavior**
It should run full speed in Firefox 61 with default 3000 cycles settings.

**Environment (please complete the following information):**
 - Windows 8.1
 - DOSBox-X master, commit a16dff8
 - Emscripten 1.38.12, Firefox 60esr

**Additional context**
I think there would be a large amount of work to completely port Dosbox-X to Emscripten, given the different model of running in a browser.

**Comment by joncampbell123** [comment]
I agree, but there's not a whole lot I can do given the way the current codebase is written. :(
